run_dir/static/js/base_b5.js

```diff
@@ -115,7 +118,7 @@ function auto_format(value, samples_table){
             value == 'no' ||
             value == 'n/a' ||
             value == 'aborted' ))){
-    returnstring = '<span class="label label-danger sentenceCase">'+value+'</span> ';
+    returnstring = '<span class="badge bg-danger sentenceCase">'+value+'</span> ';
```

Comment:
Is it possible to decrease the font-weight of the text on the badges? I think they are a bit too bold :-D

Reply:
Done!